config/default/shared-config.yaml

```diff
@@ -2,6 +2,17 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: shared-config
+  namespace: system
 data:
   VAULTHOST: ${VAULT_HOST}
   VAULTAUTHMETHOD: approle
+  VAULTINSECURETLS: "false"
+  VAULTAPPROLEROLEIDFILEPATH: "/etc/spi/role_id"
+  VAULTAPPROLESECRETIDFILEPATH: "/etc/spi/secret_id"
+  ZAPDEVEL: "false"
+  ZAPENCODER: "json"
```

Comment:
is it worth having all the defaults defined here? There are defaults in the code already.

Reply:
I think it is good for the visibility. The name of the env vars is not specified in the `*CliArg` field tags, it's just derived from the field names.

Reply:
You can see it with `go run main.go --help` and it follows simple rule. I'm concerned about having 2 places defining the same. Changing the default will require to change it in 2 places, which is not obvious.